test/artefacts/marriage-abroad/american-samoa/uk/partner_british/opposite_sex.txt

```diff
@@ -27,8 +27,6 @@ You should also check with the local authorities in American Samoa to find out i
 
 You normally have to pay fees for consular services in the local currency - these are shown in the list of [consular fees for American Samoa](/government/publications/american-samoa-consular-fees).
 
-You can pay by cash or credit card, but not by personal cheque.
-
```

Comment:
I've tried to explain why I've accepted the various regression test artefact changes in PR #2507. See https://github.com/alphagov/smart-answers/pull/2507/commits/9a91b1f78fb731d57813435ce0a6eb42c16585f2 for an explanation of these changes to American Samoa.